genmap.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+worldfile=$(python map_config/get_pkg_path.py $2)
```

Comment:
I'm not convinced this is useful. Why not let the customer just pass the .world file as an argument? If you are going to wrap it, do it for both config *and* world name entirely. However, simple is good given 1) our users are also programmers and 2) it's less to maintain/fix when it breaks. 

```
./genmap.sh <config_file> <world_file> 
```

If you want this additional help, then calling out to python seems unnecessary. Just write a function in this script to if/else on type -- but then we're defeating the purpose of a simple shell script. And if we start turning this into a command tool, we'll want --help etc.

Reply:
The consideration for separating out the config and the world was that the user might want a different map config for the same world (for e.g. higher resolution). 

Navigating to the world file could be slightly tedious/user-unfriendly (assuming no ROS/Gazebo prerequisite knowledge on the sample apps user). However, I agree that due to the helper script hard-coded into the tool, 
1. the same tool cannot be used for generic worlds
2. it could be a potential point of failure from maintenance POV